components/monitoring/grafana/development/kustomization.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
-patches:
-  - path: auto-assign-role-patch.yaml
-    target:
-      name: grafana-oauth
-      kind: Grafana
+# patches:
+#   - path: auto-assign-role-patch.yaml
+#     target:
+#       name: grafana-oauth
+#       kind: Grafana
```

Comment:
Can this be deleted instead?

Reply:
I removed the comments and added it back